### PR TITLE
moving over edited flags from v13 docs

### PR DESCRIPTION
**Pull request by hallaroo**

### CODE REVIEW

File: content/en/docs/14.0/reference/programs/vtctld.md [hunk 1]
```diff
@@ -29,92 +29,100 @@ vtctld \
...
 | -enable-query-plan-field-caching | boollean | This option fetches & caches fields (columns) when storing query plans (default true) |
 | -enable-tx-throttler | boolean | If true replication-lag-based throttling on transactions will be enabled. |
 | -enable_hot_row_protection | boolean | If true, incoming transactions for the same row (range) will be queued and cannot consume all txpool slots. |
 | -enable_hot_row_protection_dry_run | boolean | If true, hot row protection is not enforced but logs if transactions would have been queued. |
 | -enable_queries | boolean | if set, allows vtgate and vttablet queries. May have security implications, as the queries will be run from this process. |
-| -enable_realtime_stats | boolean | Required for the Realtime Stats view. If set, vtctld will maintain a streaming RPC to each tablet (in all cells) to gather the realtime health stats. |
+| -enable_realtime_stats | boolean | Required for the Realtime Stats view. If set, vtctld will maintain a streaming RPC to each tablet (in all cells) to gather the realtime health stats |
+| -enable_replication_reporter | boolean | Use polling to track replication lag |
```
Comment (deepthi) [inline content/en/docs/14.0/reference/programs/vtctld.md:74]: this flag is not relevant for vtctld.

File: content/en/docs/14.0/reference/programs/vtctld.md [hunk 2]
```diff
@@ -29,92 +29,100 @@ vtctld \
...
-| -cell | string | cell to use |
-| -ceph_backup_storage_config | string | Path to JSON config file for ceph backup storage (default "ceph_backup_config.json") |
-| -consul_auth_static_file | string | JSON File to read the topos/tokens from. |
-| -cpu_profile | string | write cpu profile to file |
-| -datadog-agent-host | string | host to send spans to. if empty, no tracing will be done |
-| -datadog-agent-port | string | port to send spans to. if empty, no tracing will be done |
-| -db-credentials-file | string | db credentials file; send SIGHUP to reload this file |
+| -binlog_use_v3_resharding_mode | boolean | True if the binlog streamer should use V3-style sharding, which doesn't require a preset sharding key column (default true) |
+| -builtinbackup_mysqld_timeout | duration | How long to wait for mysqld to shutdown at the start of the backup (default 10m0s) |
+| -builtinbackup_progress | duration | How often to send progress updates when backing up large files (default 5s) |
```
Comment (deepthi) [inline content/en/docs/14.0/reference/programs/vtctld.md:50]: I'm not sure whether the above two flags are relevant for vtctld.

File: content/en/docs/14.0/reference/programs/vtctld.md [hunk 3]
```diff
@@ -29,92 +29,100 @@ vtctld \
...
+| -catch-sigpipe | boolean | Catch and ignore SIGPIPE on stdout and stderr if specified |
+| -cell | string   | Cell to use |
+| -ceph_backup_storage_config | string   | Path to JSON config file for ceph backup storage (default "ceph_backup_config.json") |
+| -consul_auth_static_file | string   | JSON File to read the topos/tokens from |
```
Comment (deepthi) [inline content/en/docs/14.0/reference/programs/vtctld.md:54]: Same here

File: content/en/docs/14.0/reference/programs/vtctld.md [hunk 4]
```diff
@@ -29,92 +29,100 @@ vtctld \
...
+| -cpu_profile | string   | Deprecated: use '-pprof=cpu' instead |
+| -datadog-agent-host | string   | Host to send spans to; if empty, no tracing will be done |
+| -datadog-agent-port | string   | Port to send spans to; if empty, no tracing will be done |
 | -db-credentials-server | string | db credentials server type (use 'file' for the file implementation) (default "file") |
 | -dba_idle_timeout | duration | Idle timeout for dba connections (default 1m0s) |
 | -dba_pool_size | int | Size of the connection pool for dba connections (default 20) |
-| -disable_active_reparents | boolean | if set, do not allow active reparents. Use this to protect a cluster using external reparents. |
+| -disable_active_reparents | boolean | If set, do not allow active reparents. Use this to protect a cluster using external reparents |
```
Comment (deepthi) [inline content/en/docs/14.0/reference/programs/vtctld.md:61]: I believe this is relevant only to vttablet, not vtctld.

File: content/en/docs/14.0/reference/programs/vtctld.md [hunk 5]
```diff
@@ -29,92 +29,100 @@ vtctld \
...
-| -log_rotate_max_size | uint | size in bytes at which logs are rotated (glog.MaxSize) (default 1887436800) |
-| -logtostderr | boolean | log to standard error instead of files |
+| -log_backtrace_at | value    | When logging hits line file:N, emit a stack trace |
+| -log_dir | string   | If non-empty, write log files in this directory |
+| -log_err_stacks | boolean | Log stack traces for errors |
+| -log_rotate_max_size | uint     | Size in bytes at which logs are rotated (glog.MaxSize) (default 1887436800) |
+| -logtostderr | boolean | Log to standard error instead of files |
 | -replication_connect_retry | duration | how long to wait in between replica reconnect attempts. Only precise to the second. (default 10s) |
-| -mem-profile-rate | int | profile every n bytes allocated (default 524288) |
+| -master_connect_retry | duration | Deprecated, use -replication_connect_retry (default 10s) |
```
Comment (deepthi) [inline content/en/docs/14.0/reference/programs/vtctld.md:122]: should be deleted.

File: content/en/docs/14.0/reference/programs/vtctld.md [hunk 6]
```diff
@@ -159,16 +170,19 @@ vtctld \
...
-| -s3_backup_force_path_style | boolean | force the s3 path style |
-| -s3_backup_log_level | string | determine the S3 loglevel to use from LogOff, LogDebug, LogDebugWithSigning, LogDebugWithHTTPBody, LogDebugWithRequestRetries, LogDebugWithRequestErrors (default "LogOff") |
-| -s3_backup_server_side_encryption | string | server-side encryption algorithm (e.g., AES256, aws:kms) |
-| -s3_backup_storage_bucket | string | S3 bucket to use for backups |
-| -s3_backup_storage_root | string | root prefix for all backup-related object names |
-| -s3_backup_tls_skip_verify_cert | boolean | skip the 'certificate is valid' check for SSL connections |
+| -relay_log_max_items | int      | Maximum number of rows for VReplication target buffering (default 5000) |
+| -relay_log_max_size | int      | Maximum buffer size (in bytes) for VReplication target buffering. If single rows are larger than this, a single row is buffered at a time (default 250000) |
+| -remote_operation_timeout | duration | Time to wait for a remote operation (default 30s) |
+| -replication_connect_retry | duration | How long to wait in between replica reconnect attempts. Only precise to the second (default 10s) |
```
Comment (deepthi) [inline content/en/docs/14.0/reference/programs/vtctld.md:176]: Flag above is not relevant.

File: content/en/docs/14.0/reference/programs/vtctld.md [hunk 7]
```diff
@@ -159,16 +170,19 @@ vtctld \
...
-| -s3_backup_aws_region | string | AWS region to use (default "us-east-1") |
-| -s3_backup_aws_retries | int | AWS request retries (default -1) |
-| -s3_backup_force_path_style | boolean | force the s3 path style |
-| -s3_backup_log_level | string | determine the S3 loglevel to use from LogOff, LogDebug, LogDebugWithSigning, LogDebugWithHTTPBody, LogDebugWithRequestRetries, LogDebugWithRequestErrors (default "LogOff") |
-| -s3_backup_server_side_encryption | string | server-side encryption algorithm (e.g., AES256, aws:kms) |
-| -s3_backup_storage_bucket | string | S3 bucket to use for backups |
-| -s3_backup_storage_root | string | root prefix for all backup-related object names |
-| -s3_backup_tls_skip_verify_cert | boolean | skip the 'certificate is valid' check for SSL connections |
+| -relay_log_max_items | int      | Maximum number of rows for VReplication target buffering (default 5000) |
+| -relay_log_max_size | int      | Maximum buffer size (in bytes) for VReplication target buffering. If single rows are larger than this, a single row is buffered at a time (default 250000) |
```
Comment (deepthi) [inline content/en/docs/14.0/reference/programs/vtctld.md:174]: @rohit-nayak-ps @mattlord do these two flags apply to vtctld?

File: content/en/docs/14.0/reference/programs/vtctld.md (hunk 1 above)
Comment (hallaroo) [reply content/en/docs/14.0/reference/programs/vtctld.md:74]: That might be an issue. I'm copying over the flags from the recently updated v13 docs:

https://github.com/vitessio/website/blob/prod/content/en/docs/13.0/reference/programs/vtctld.md?plain=1#L61

File: content/en/docs/14.0/reference/programs/vtctld.md (hunk 2 above)
Comment (hallaroo) [reply content/en/docs/14.0/reference/programs/vtctld.md:50]: Adding in the links to the v13 docs for the built in flags:

https://github.com/vitessio/website/blob/prod/content/en/docs/13.0/reference/programs/vtctld.md?plain=1#L48
https://github.com/vitessio/website/blob/prod/content/en/docs/13.0/reference/programs/vtctld.md?plain=1#L49

File: content/en/docs/14.0/reference/programs/vtctld.md (hunk 4 above)
Comment (hallaroo) [reply content/en/docs/14.0/reference/programs/vtctld.md:61]: Also in the v13 docs:

https://github.com/vitessio/website/blob/prod/content/en/docs/13.0/reference/programs/vtctld.md?plain=1#L57

File: content/en/docs/14.0/reference/programs/vtctld.md (hunk 5 above)
Comment (hallaroo) [reply content/en/docs/14.0/reference/programs/vtctld.md:122]: Also in the v13 docs:

https://github.com/vitessio/website/blob/prod/content/en/docs/13.0/reference/programs/vtctld.md?plain=1#L99

File: content/en/docs/14.0/reference/programs/vtctld.md (hunk 6 above)
Comment (hallaroo) [reply content/en/docs/14.0/reference/programs/vtctld.md:176]: Also in the v13 docs:

https://github.com/vitessio/website/blob/prod/content/en/docs/13.0/reference/programs/vtctld.md?plain=1#L115

File: content/en/docs/14.0/reference/programs/vtctld.md (hunk 7 above)
Comment (hallaroo) [reply content/en/docs/14.0/reference/programs/vtctld.md:174]: Also in the v13 docs:

https://github.com/vitessio/website/blob/prod/content/en/docs/13.0/reference/programs/vtctld.md?plain=1#L113

**Comment by hallaroo** [comment]
@deepthi - We might need to go over the v13 docs again based on the comments you left above. I can hold off on the v14 docs for now until we get the v13 fully ironed out, so that I don't copy over any other incorrect flags.

**Pull request closed without merging**